web/api/controller/market_controller.py

```
@@ -0,0 +1,28 @@
+from fastapi import APIRouter
+from web.api.request.inference_request import MarketNewsRequest, MarketSectorRequest
+
+router = APIRouter(tags=["Market Intelligence"])
+
+
+@router.post("/news/analyze", summary="특정 뉴스 심층 분석")
+async def analyze_news_deep(req: MarketNewsRequest):
+    # URL이 있으면 크롤러 동작, 없으면 content 사용
+    target_content = req.content
```

Ruff (0.14.5)

Local variable `target_content` is assigned to but never used

Remove assignment to unused variable `target_content`

(F841)